extension/src/manifest.json

```diff
@@ -7,13 +7,14 @@
     "browser_action": {
         "default_icon": "icon.png"
     },
-    "web_accessible_resources": [
-        "js/inject.js"
-    ],
     "background": {
         "scripts": ["js/lib/adapter.js", "js/lib/socket.io.js", "js/background.js"],
         "persistent": true
     },
+    "content_scripts": [{
+        "matches": ["<all_urls>"],
```

Comment:
What is the purpose of this?

Reply:
https://stackoverflow.com/questions/16096482/what-does-http-https-and-all-urls-mean-in-the-context-of-ch